src/main/resources/site/parts/highchart/Highchart.jsx

```diff
@@ -102,18 +103,29 @@ function Highchart(props) {
     )
   }
 
-  function renderHighchartsSources(highchart) {
+  function renderHighchartsSource(sourceLink) {
     return (
-      <Col className='col-12'>
-        {highchart.footnoteText ? <Row className='col-12 footnote'>{highchart.footnoteText}</Row> : null}
-        {highchart.creditsEnabled ? (
-          <Row className='mt-4 mt-md-5 highcharts-source'>
-            <Col className='col-12 fw-bold mb-0'>{props.phrases.source}:</Col>
-            <Col className='col-12'>
-              <Link href={highchart.creditsHref}>{highchart.creditsText}</Link>
-            </Col>
+      <Row>
```

Comment:
Add key to avoid warning in vs code

Reply:
I don't get any warnings in VS code. Guess there is some config or extension I am missing, can you show me?